sync/class.jetpack-sync-functions.php

```diff
@@ -55,16 +55,16 @@ public static function file_system_write_access() {
 
 		$filesystem_method = get_filesystem_method();
 		if ( $filesystem_method === 'direct' ) {
-			return 1;
+			return '1';
```

Comment:
That is how we store it on .com. It is a bit of a legacy thing. Where we we didn't want to pass false though the function because it was a mock function. Which if you back false it would try to grab the value from the DB instead of shortcutting it and returning a 1 or 0. Let me know if we want to just return true or false.